packages/core/package.nls.json

```diff
@@ -142,8 +142,8 @@
     "AWS.command.refreshAwsExplorer": "Refresh Explorer",
     "AWS.command.refreshCdkExplorer": "Refresh CDK Explorer",
     "AWS.command.cdk.help": "View CDK Documentation",
-    "AWS.command.ec2.openTerminal": "Open terminal to EC2 instance...",
-    "AWS.command.ec2.openRemoteConnection": "Connect to EC2 instance in New Window...",
+    "AWS.command.ec2.openTerminal": "Connect Terminal to EC2 Instance...",
+    "AWS.command.ec2.openRemoteConnection": "Connect VS Code to EC2 Instance...",
```

Comment:
Normally we would avoid hardcoding the IDE name. But since vscode remote-ssh requires VSCode, this works for now.